### DISCUSSION TITLE
No complete function with 1.0.0

### DISCUSSION THREAD

**Discussion by Textleiche** [calibx/msfsdeck#91] [General]
Windows10pro, with all updates
Loupedeck Live latest update installed, firmware up to date
FSUPic uninstalled (clean, with RevoUninstaller)
PlugIn 1.0.0 installed
Reboot was also done.

Only the page (6) with e.g. avionics on/off and mouse wheel works. Page 1. is completely dead! No feedback and no function. If I press a button/wheel, the MSFS is called up in the task bar.

Any ideas?

But thanks anyway for the great work. It has given me a lot of pleasure so far.

**Comment by calibx** [comment]
Hi,

What do you have on the pages ?
Can you make another profil with only one encoder ?

**Reply by Textleiche** [reply]
I only open the MSFS profile, no other changes. This has always worked well for me so far. The problem is that only page 6 works, and the buttons 1/2 for pause switch. The most important page 1 (AP, Lights,Status) does not.
FSUPIC was uninstalled right at the beginning, or do I still need it?
The Loupedeck software has already been cleanly uninstalled (RevoUninstaller) and reinstalled. Nothing happens.

Thanks for the feedback.

**Comment by calibx** [comment]
Can you try loading this profile (after renaming it to default.lp5) with only 2 controls : connect and beacon ? [default.lp5.txt](https://github.com/calibx/msfsdeck/files/8644782/default.lp5.txt)
Then go to MSFS, clic on "beacon" on the loupedeck, the other input should display "connected".

Can you confirm the version of Loupedeck (5.1.xx) in the about menu ?

**Reply by Textleiche** [reply]
1. import "default.ip5
Rename internally to "defaultMSFS" because the IP4 is already called MSFS.
2. button/display from "MSFSPluginPackage-1.0.0.lplug4" debug/connect added
3. cessna citation on runway (EDDL)

Button "Beacon" changes its status display, but "Trying to connect..." does not change. Nothing changes in the light panel of the Cessna. 

It seems to me that the response time of the Loupedeck is getting slower! I also noticed this with the IP4, as there is more implemented. It goes almost to the point of freezing. With my "Windows" profile everything is ok again.

- Loupedeck Software 5.1.0.12043 (Firmware 0.2.5), latest for me
- MSFS without mods
- FSUPIC is not installed (clean deinstall: reg. and files)
- No other software connected to the MSFS
- Win10pro x64, latest
- CPU 5800x and GPU 5700xt from AMD, 32GB RAM

Loupedeck works normally with my "Windows profile", here I only use date, time, volume, OBS and keyboard shortcuts for a CAD program.

Just ideas: 
- my system is set to German language. Maybe a language pack is used?
- Are there any "runtimes" missing on my system, e.g. VC+ or .net (4.8 is installed)? Nothing is reported? It may be that you have packages available with e.g. VisualStudio and I don't have them. 
- other "dependencies" on software packages?
- Are there still remains of FSUPIC? I have installed FSUPIC with RevoUninstaller, including registry and file scan -> clean deinstallation.

Thanks for looking after it, it's a great package.

**Comment by calibx** [comment]
"Trying to connect" means that the plugin is waiting datas from MSFS. It is the first time I see that problem.
Are you using a specific firewall ?
FSUIPC shouldn't prevent anything, nor language or libs : all should be in the zip.

**Reply by Textleiche** [reply]
Uninstalled everything again today and reinstalled (MSFSPluginPackage-1.0.0.1.lplug4). It works. I think that I was the problem or that the Loupedeck software did not correctly adopt the new lplug4.

Thank you for your answers and efforts.

**Reply by calibx** [reply]
Great news !!
